### PR TITLE
Add structured encrypt example (GEA-11385)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This is a quick example about how you use Pangea Go SDK, set up and run it.
 
 ## Setup
 
-Set up environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_AUDIT_TOKEN` and `PANGEA_DOMAIN` with your project token configured on Pangea User Console (token should have access to Audit service [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your pangea domain.
+Set up environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_AUDIT_TOKEN` and `PANGEA_DOMAIN` with your project token configured on Pangea User Console (token should have access to Audit service [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your Pangea domain.
```
Comment (pangea-andrest) [inline examples/README.md:7]: This readme talk just about `Audit` service. We should make it more generic due to it's in examples root directory

**Review by pangea-andrest — DISMISSED** [review]

File: examples/README.md (hunk 1 above)
Comment (kenany) [reply examples/README.md:7]: Agreed. I'm trying to now but it's difficult when there are so many different environment variables. `PANGEA_DOMAIN` is a constant one, that's fine, and sure we can say there is one per service in the form of `PANGEA_{SERVICE_NAME}_TOKEN`, but then there are very specific variables like `PANGEA_AUDIT_MULTICONFIG_TOKEN`, `PANGEA_AUDIT_CONFIG_ID`, `PANGEA_AUDIT_CUSTOM_SCHEMA_TOKEN`, `PANGEA_REDACT_MULTICONFIG_TOKEN`, ...

Comment (kenany) [reply examples/README.md]: Alright I simplified it all, take a look.

File: examples/vault/structured_encrypt.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+	ctx := context.Background()
+
+	// First create an encryption key, either from the Pangea Console or
+	// programmatically as below.
+	generateInput := &vault.SymmetricGenerateRequest{
+		Algorithm: vault.SYAaes256_cfb,
+		Purpose:   vault.KPencryption,
+		CommonGenerateRequest: vault.CommonGenerateRequest{
+			Name: "any unique name",
```
[the remark this answers is not in the document]
Comment (kenany) [reply examples/vault/structured_encrypt.go]: Okay, done.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**